ci/values.yaml

```diff
@@ -66,6 +66,7 @@ config:
         http:
           endpoint: 0.0.0.0:4318
   exporters:
+    debug: null
```

Comment:
Could you add a comment on when we can remove this (I think once we merge upstream should be good ?)

Reply:
Done